### PR TITLE
feat(service-portal): Add clients for directorate of health apis

### PR DESCRIPTION
**Pull request by disaerna**
# Clients - Directorate of health

## What
Adding two new clients to health directorate folder.
All api's coming from the same source. May change later

## Why

Part of adding heilsuvera to mínar síður

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

## Summary by CodeRabbit

- **New Features**
	- Introduced new client configurations for vaccinations and organ donation services, enhancing health-related functionalities.
	- Added new services to manage organ donation and vaccination-related functionalities.
	- Launched OpenAPI specifications for Vaccination and Organ Donor APIs to streamline data access.

- **Bug Fixes**
	- Improved path resolutions in configuration files for better module organization.

- **Documentation**
	- Updated export modules to improve clarity and accessibility of health-related services and configurations.

- **Chores**
	- Restructured configuration factories to enhance validation and service management.

### CODE REVIEW

**Comment by disaerna** [comment]
@coderabbitai review

**Review by Toti91 — APPROVED** [review]
Application-system LGTM

File: libs/clients/health-directorate/src/lib/clients/vaccinations/vaccinations.service.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  public async getVaccinations(
+    auth: Auth,
+    input: MeVaccinationControllerGetVaccinationsForDiseasesRequest,
+  ): Promise<Array<DiseaseVaccinationDto> | null> {
+    const vaccines = await this.vaccinationsApiWithAuth(auth)
+      .meVaccinationControllerGetVaccinationsForDiseases(input)
+      .catch(handle404)
+
+    if (!vaccines) {
+      return null
```
Comment (thordurhhh) [inline libs/clients/health-directorate/src/lib/clients/vaccinations/vaccinations.service.ts:29]: Do we need logging here and on l:44 if there's a null response?

Something like this, maybe with more info:

```
      this.logger.warn('No vaccines returned', {
        category: LOG_CATEGORY,
      })
```

**Review by thordurhhh — APPROVED** [review]
Nice. Just one minor question.

**Review by Ballioli — APPROVED** [review]
LGTM

File: libs/clients/health-directorate/src/lib/clients/vaccinations/vaccinations.service.ts (hunk 1 above)
Comment (disaerna) [reply libs/clients/health-directorate/src/lib/clients/vaccinations/vaccinations.service.ts:36]: Updated - also adder warning in organ donation as they should not be null either :) 
Can you re-review 😇 ?

**Review by snaerseljan — APPROVED** [review]
LGTM!

**Pull request merged**